app/assets/stylesheets/home.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Home controller here.
```

Comment:
Do we need to add this file? Can we remove it for now, since it is empty?

Reply:
Agree. Done